### ISSUE TITLE
[Feature Request] - Automatically accept files from trusted devices

### ISSUE THREAD

**Issue by IAmOrion** [LANDrop/LANDrop#119]
I'm using LANDrop on a few of my devices.  I work using a Macbook pro, but also have access to a PC (outside of working hours).
The computers are located in different rooms (well, the macbook is obv portable so can work from any room so to so speak).
I've sometimes come across something (or remembered something I wanted to copy to my PC) when I'm working and in a different room from the PC.  

It would be awesome if I was able to send files to my PC knowing that they'll be there ready when I get around to getting onto my PC.  The problem at the moment is that you have to confirm receipt which is frustrating (albeit just a minor inconvenience).  It would be good if you could add devices (or be prompted to save incoming device connection to always accept) to allow certain devices to be able to send without having to accept them - they would be automatically downloaded.

**Comment by snottzoid** [comment]
I repeat the same idea. I have downloaded the source code and I am installing Qt and libsodium for Windows, but I'm sure the auther would be much faster at this. If I am able to figure it out, my version will simply accept any file automatically, and not have a list of trusted senders. (that is more than I want to bite off right now)

**Comment by im-ichat** [comment]
i would agree to your idea that it would be verry usefull to have such feature as, i recently got a windows laptop again after working on a  macbook for over a year. 

Not having the apple filesend thingy was a bit of a hassle untill i found out about landrop. 
It really is an awsome app. Being able to automagically send files to another device as soon as contact is made (ie the same wifinetwork is joined) would be perfect. 
However,  as a handshake is usually generated on the fly and is easily Human Checkable,  generating some kind of white-list might not be so easy as reusing a password multible times would set you up for people intercepting it and using it to send harmfull files to your devices. 
So for this to work  you might need some sort of  OTP-generator on both devices to be able to still be safe from users sending (and auto accepting) files onto your computer, but especially on mobile devices OTP isn't allways the most batery friendly. 

this for context.
